sdk/metricsadvisor/ci.yml

```diff
@@ -29,7 +29,3 @@ extends:
   parameters:
     ServiceDirectory: metricsadvisor
     TestProxy: true
-    Artifacts:
-    - name: azure-ai-metricsadvisor
-      safeName: azureaimetricsadvisor
-
```

Comment:
Will this silently work? I have no idea

Reply:
Looks like it does: https://dev.azure.com/azure-sdk/public/_build/results?buildId=4083031&view=logs&j=8e5196d1-9c2a-56bb-41fd-e329e85c48fa&t=e2ecfcdb-d2d5-5b7d-c312-a514a9344a8c&l=13